collections/_article/release-candidate-godot-4-1-rc-2.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Release candidate: Godot 4.1 RC 2"
+excerpt: "A few more regressions were fixed, and we should now be ready for the Godot 4.1 release. Help us confirm that by testing RC 2!"
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/release-candidate-godot-4-1-rc-2.webp
+image_caption_title: "Skies above the Great War"
+image_caption_description: "A game by Puntigames"
+date: 2023-06-30 09:00:00
+---
+
+Godot 4.1 is shaping up to be ready [right on schedule](/article/release-management-4-1). Over the course of the beta testing phase, and with our first [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) a few days ago, we identified and fixed a number of regressions, i.e. recently introduced bugs that would worse the experience compared to Godot 4.0. Now we're fairly confident that we've handle most of those, and the remaining issues we're tracking will likely be deferred to a fix in future 4.1.x maintenance releases.
+
+That is to say, this RC 2 should be the one – if no major regression is identified in coming days, we'll go ahead and release 4.1 stable, and reopen the development cycle for 4.2, which is scheduled for a release in November 2023. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – we might triage not fix them for the imminent 4.1 release, but they may be prioritized to fix early on in the 4.2 development cycle, and backported to 4.1.x.
```

Comment:
```suggestion
That is to say, this RC 2 should be the one – if no major regression is identified in coming days, we'll go ahead and release 4.1 stable, and reopen the development cycle for 4.2, which is scheduled for a release in November 2023. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – we might decide not to fix them for the imminent 4.1 release, but they can be prioritized to be fixed early on in the 4.2 development cycle, and backported to 4.1.x.
```

This doesn't read right, but you can go with something else rather my suggestion.